cmd/webhook/app/options/options.go

```diff
@@ -2,8 +2,12 @@ package options
 
 import (
 	"github.com/spf13/pflag"
+	"k8s.io/component-base/cli/globalflag"
 
 	"github.com/karmada-io/karmada/pkg/sharedcli/profileflag"
+	// ensure libs have a chance to globally register their flags
+	_ "github.com/karmada-io/karmada/pkg/webhook/clusterpropagationpolicy"
+	_ "github.com/karmada-io/karmada/pkg/webhook/propagationpolicy"
```

Comment:
I have a doubt about whether the import path needs to be changed to `github.com/karmada-io/karmada/pkg/webhook/defaulttolerationseconds`.

Reply:
Sorry, my mistake here 🤣